pkg/crc/machine/machine.go

```diff
@@ -170,6 +170,22 @@ func Start(startConfig StartConfig) (StartResult, error) {
 			return *result, errors.Newf("Error loading host: %v", err)
 		}
 
+		vmState, err := host.Driver.GetState()
+		if err != nil {
+			result.Error = err.Error()
+			return *result, errors.Newf("Error getting the state for host: %v", err)
+		}
+		if vmState == state.Running {
+			openshiftVersion := crcBundleMetadata.GetOpenshiftVersion()
```

Comment:
I see `bundleName, crcBundleMetadata, err = getBundleMetadataFromDriver(host.Driver)` below this block, is this working as expected? (I did not look very closely nor run it for a test)

Reply:
@cfergeau yes `crcBundleMetadata` is used as part of getting openshift version, so `IbundleName, crcBundleMetadata, err = getBundleMetadataFromDriver(host.Driver)` should be above that block.